MechJeb2/FuelFlowSimulation.cs

```diff
@@ -263,15 +263,22 @@ private bool AllowedToStage()
                             return false;
                         }
 
-                        if (n.ContainsResources(burnedResources.value))
+                        foreach (int id in burnedResources.value)
```

Comment:
this had to be changed to loop over each resource, uniquely checking if an activeengine could draw that resource from the part inspected to see if it could be dropped.